README.md

```diff
@@ -3,13 +3,58 @@ Welcome to `dlings`. This project aims to teach you how to program in D.
 If the name doesn't sound strange, it's because you might have heard of [rustlings](https://github.com/rust-lang/rustlings) or [ziglings](https://github.com/ratfactor/ziglings). In fact, this is **heavily** based on [rustlings](https://github.com/rust-lang/rustlings). If you're familiar with how [rustlings](https://github.com/rust-lang/rustlings) work, you will have no trouble getting started with `dlings`.
 
 # Getting started
+## Dependencies
+* [argparse](https://code.dlang.org/packages/argparse)
+* [fswatch](https://code.dlang.org/packages/fswatch)
+
+## Clone the repository
 ```sh
 git clone git@github.com:iK4tsu/dlings.git
 cd dlings
+```
+
+## Using DUB
+```sh
 dub build -b release
 ./dlings --help
 ```
 
+## Installing with Meson
+__Note:__ DUB prioritizes `dmd` when building dependencies. Since Meson is
+using DUB's local repository to find dependencies, you must ensure both
+Meson and DUB are using the same compiler.
```

Comment:
```suggestion
__Note:__ DUB prioritizes `dmd` when building dependencies. Since Meson is
using DUB's local repository to find dependencies, you must ensure both
Meson and DUB are using the same compiler. This can be done by passing
the `--compiler` argument to DUB, and setting the `DC` environment variable
when invoking Meson. Pay attention to these options below.
```

Reply:
Fixed in 850b144577817e4a9dc1673b4b521f665dcc43c1